### DISCUSSION TITLE
Button shortcut for 'disable TX'

### DISCUSSION THREAD

**Discussion by ON7MHZ** [egzumer/uv-k5-firmware-custom#320] [Ideas]
Since this (amazing) transceiver is (unfortunately) not legal in all countries ...  a small question/suggestion

Can it be made possible to 'disable all TX' (now only available via hidden menu 61) to be allocated to a button (via menu 23-27)?
This way the unit can become RX only in the blink of an eye if necessary.

Thanks for taking this into consideration
73

**Comment by gartnl** [comment]
That would only be usefull if it was a one-way setting: you can disable TX with one button, but have to enable it with the current method.
In that case it would not give much advantage over the current method.
But: as a HAM you should know that TX outside HAM bands is bad and outside the default VHF/UHF bands is unusable. Actually there is no need to have 'all TX' enabled at all except for testing with a dummyload in your shack.
If it were up to me I would remove the 'all TX' option completely.

**Reply by ErikS-web** [reply]
And with Chirp-EGZUMER driver V0.5 you have the opportunity to do 

* _**Added Duplex Out as a way to disable TX in a channel**_

This way you can have a memory channel, which cannot be TX-used by _mistake._

**Reply by ON7MHZ** [reply]
Its not about TX outside ham bands (nothing interesting there anymore), but about countries where this unit is not allowed to be used at all.
So enabling TX again via menu settings is not a problem, would rather be to use it as an 'emergency button' to make this an RX only device in case its necessary. But hey, its only a suggestion ;)

**Comment by ErikS-web** [comment]
Can be done within 60 seconds, and _practise to go faster_ ;-)

`Hidden menu` is activated by holding `PTT` + `SIDE BUTTON 1` while turning on the radio and than Release All Keys.

61. `F Lock` - sets the TX frequency band plan.

select option **DISABLE ALL** - disables TX on all frequencies
